eclipse-collections-api/src/main/java/org/eclipse/collections/api/factory/set/sorted/MutableSortedSetFactory.java

```diff
@@ -21,6 +21,8 @@
      */
     <T> MutableSortedSet<T> empty();
 
+    <T> MutableSortedSet<T> empty(Comparator<? super T> comparator);
```

Comment:
This will require a `since` tag of 12.0 since it is an API change without a default implementation.